todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/BasePresenter.java

```diff
@@ -20,6 +20,6 @@
 
     void subscribe();
 
-    void unsubscribe();
+    void unSubscribe();
```

Comment:
revert. Unsubscribe is a word.

Reply:
Oh well, I have only been misusing the word since.. ever :). Will revert.